requirements.txt

```diff
@@ -11,3 +11,4 @@ pywin32; sys_platform == 'win32'
 py2-ipaddress; sys_platform == 'win32' and python_version == '2'
 certifi; sys_platform == 'win32'
 zope.interface>=4.4.2
+distro; platform_system != 'Windows' and platform_system != 'Microsoft' and platform_system != 'Darwin'
```

Comment:
Not sure if this condition is good.
What do you think @cas-- ?

Reply:
I would simplify to `platform_system != 'Windows' and platform_system != 'Darwin'`. I don't think the Microsoft check is required these days.

I do wonder what happens if try to install distro on those OS but it's fine with the check for now.

An amusing alternative would be `'win' not in sys_platform` since it would cover `cygwin`, `win32` and `darwin` but a bit too obscure ;)

Reply:
I installed `distro` on windows successfully, but all the functions return empty string to something like it.

And I changed the line to the obscure way :wink: